### ISSUE TITLE
Create feature list

### ISSUE THREAD

**Issue by guido4000** [ryansolid/solid-ts-webpack#3]
@ryansolid in the main repo you mention, that there is a lot missing in this baseline setup.
https://github.com/ryansolid/solid/issues/144#issuecomment-636517714

Maybe it would make sense to create a list with completed and missing features. This would be great for transparency and it would be easier for others to contribute to it.

**Comment by ryansolid** [comment]
I agree. That's part of the problem though. Since I don't even know what isn't here. Which is part of why I haven't tried to make something that is full-featured. The other problem is there are many opinionated options here that there are no clear winners. So it takes a certain amount of really weighing the options.

Some obvious things that are lacking:
* idiomatic CSS setup
* testing framework
* better cli messages
* public folder aliasing for production builds
* PWA requirements (service worker/manifest)
* routing

Every time someone asks a question about Create Solid I learn about a new feature in it I wasn't even aware of. Some of the premade skeletons around Solid have been getting more fully-featured.
